README.md

```diff
@@ -34,55 +34,68 @@ This proposal pursues the third option, as we expect it to lead to the best deve
 
 Module attributes have to be made available in several different contexts. This section contains one possible syntax, but there are other options, discussed in [#6](https://github.com/littledan/proposal-module-attributes/issues/6).
 
-Here, a key-value syntax is used, with the key `type` used as an example indicating the module type. Such key-value syntax can be used in various different contexts. Another option is to have just a single string (discussion in [#11](https://github.com/littledan/proposal-module-attributes/issues/11)).
+Here, a single string is permitted to describe a single module attribute, as discussed in [#12](https://github.com/littledan/proposal-module-attributes/issues/12).
 
-The JavaScript standard would basically be responsible for passing the options bag up to the host environment, which could then decide how to interpret it. Issues [#24](https://github.com/littledan/proposal-module-attributes/issues/24) and [#25](https://github.com/littledan/proposal-module-attributes/issues/25) discuss the Web and Node.js feature and semantic requirements respectively, and issue [#10](https://github.com/littledan/proposal-module-attributes/issues/10) discusses how to allow different JavaScript environments to have interoperability.
+Although unspecified in the module attributes proposal, this intention of the proposal champions is that this string would be interpreted as the module type, as shown in the following examples.
 
-### import statements
+The JavaScript standard would basically be responsible for passing the string up to the host environment, which could then decide how to interpret it. Issues [#24](https://github.com/littledan/proposal-module-attributes/issues/24) and [#25](https://github.com/littledan/proposal-module-attributes/issues/25) discuss the Web and Node.js feature and semantic requirements respectively, and issue [#10](https://github.com/littledan/proposal-module-attributes/issues/10) discusses how to allow different JavaScript environments to have interoperability.
 
-The ImportDeclaration would allow any arbitrary attributes after the `with` keyword.
+### import statements
 
-For example, in some environments a `type` key could be mandatory to load a module that isn't JavaScript (json, css, html, ...).
+The ImportDeclaration would allow a string provided at the end of an import statement with the `as` keyword.
 
 ```js
-import json from "./foo.json" with type: "json";
+import json from "./foo.json" as "json";
 ```
 
+The host environment would determine the interpretation of `"json"`. In the Web and similar environments, `as "json"` would be required to load JSON modules, whereas no `as` syntax would be used for JavaScript modules.
+
 ### dynamic import()
 
-The `import` function will allow to provide arbitrary attributes as its second arguments.
+The `import()` pseudo-function would allow the string as its second argument.
 
 ```js
-import("foo.json", { type: "json" })
+import("foo.json", "json")
```

Comment:
It seems to be consitent with the ImportDeclaration syntax, allowing both a string and an object (in the future), but It doesn't look as descriptive to me.

what about using the same semantics than the worker instantion, `import("foo.json", { as: "json" })`?

Reply:
Are there other parameters we might want to send to `import()` that would not make sense as module attributes?

Reply:
I don't know, my only argument is that it doesn't look as obvious as an ObjectLiteral with an `as` key, but I don't feel very strong about that.

Reply:
Yeah, I agree that the placement-based syntax isn't so obvious. But, I think if we add object attributes later, it will be weird to have to put them nested inside this `{as:}` construct.

Reply:
+1 to @littledan's take here. If we want to use the object form I think we should be consistent with the name of the field rather than `as`